build.gradle

```diff
@@ -197,7 +198,7 @@ def setupTasks(SourceSet sourceSet) {
     tasks.named(sourceSet.processResourcesTaskName, ProcessResources) {
         setGroup('process resources')
         duplicatesStrategy = DuplicatesStrategy.FAIL
-        var versionProperties = ['version': mod_version, 'mc_version': minecraft_version_range, 'forge_version': forge_version_range, 'loader_version': loader_version_range,
+        var versionProperties = ['file': ['jarVersion': mod_version], 'mc_version': minecraft_version_range, 'forge_version': forge_version_range, 'loader_version': loader_version_range,
```

Comment:
Is there a reason for changing how this defines the version? I am not strongly opposed to it, I just want to understand it.